src/uct/ib/base/ib_device.h

```diff
@@ -53,6 +53,7 @@
 #define UCT_IB_SITE_LOCAL_PREFIX          be64toh(0xfec0000000000000ul) /* IBTA 4.1.1 12b */
 #define UCT_IB_SITE_LOCAL_MASK            be64toh(0xffffffffffff0000ul) /* IBTA 4.1.1 12b */
 #define UCT_IB_SITE_LOCAL_FLID_MASK       be64toh(0xffffffff00000000ul) /* site-local + flid */
+#define UCT_IB_OPENIB_OUI                 0x001405
```

Comment:
is it part of GID/GUID? let's add to the name

Reply:
it's a specific value we look for in GID.
do you propose something like `UCT_IB_GID_OPENIB_OUI`? 
imo, this looks a bit confusing because it is not a gid, but some specific ID

Reply:
so can we provide some documentation for it?

Reply:
removed this one and used `IB_OPENIB_OUI` from `mad.h`. Also added comment